### ISSUE TITLE
Supporting URL is outdated

### ISSUE THREAD

**Issue by dr29bart** [SeleniumHQ/selenium#6636]
## Meta -
Java: Selenium 3.14.0

StaleElementReferenceException message contains [supporting link](http://seleniumhq.org/exceptions/stale_element_reference.html):
```
org.openqa.selenium.StaleElementReferenceException: stale element reference: element is not attached to the page document
  (Session info: chrome=70.0.3538.77)
  (Driver info: chromedriver=2.43.600210 (68dcf5eebde37173d4027fa8635e332711d2874a),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
```
which leads to main page instead of the [actual SERE page](https://www.seleniumhq.org/exceptions/stale_element_reference.jsp)

Please, update protocol and the extension.

**Comment by cgoldberg** [comment]
looks like if we add `www.` in the domain, it properly redirects to the right place.  see: #6639

**Comment by cgoldberg** [comment]
fixed with 700f5b1ea6fef50250cdc2f7fae17f99ba1fe28e
